### ISSUE TITLE
i18n WG Meeting - Feb 28

### ISSUE THREAD

**Issue by alexandrtovmach** [nodejs/i18n#217]
### Date & Time
* 6:00pm UTC on Feb 28, 2020

### Links
* [Zoom link](https://zoom.us/j/307660555)
* [Google Docs Minutes](https://docs.google.com/document/d/11xXdYFJi-hslaT2qvBkQdC9X9ICIFSCCaNvJDtmLBHo/edit?usp=sharing)

#### l10n
* Crowdin demo: https://github.com/nodejs/i18n/issues/216
* Localization best practices guide, #214
* Update our Guides, #210
* Create Crowdin Glossary, #81, #91

#### i18n
* Unblock translation process, [#2906](https://github.com/nodejs/nodejs.org/issues/2906)
* Publish a module to npm, #92
* Website content to be translated, #72
* Automatic approve and merge translation PRs, #161

#### Intl
* Onboarding new Intl contributors (not yet an issue)
  * What kind of help is needed the most in Intl right now?

**Note: Please feel free to add any issues you'd like to discuss today.** 

### Invited Members
* @nodejs/i18n
* @nodejs/website-redesign
* @nodejs/community-committee
* @nodejs/tsc
* @nodejs/website

**Comment by alexandrtovmach** [comment]
@obensource seems busy, so I took the responsibility of creating this issue. Please mention any issues/points to discuss today.

**Comment by maddhruv** [comment]
I won't be able to join even this time, stuck in some other meetings
Might join in the middle

**Comment by obensource** [comment]
@alexandrtovmach thanks for creating this issue, I really appreciate it! 🙏

@nodejs/i18n – on that note, I hope everyone feels like they can pitch in anywhere (like @alexandrtovmach, and @maddhruv a couple weeks ago) if something needs to be done. We're all in this together! ♥️

In this case, it would be nice to automate opening and closing our WG meeting issues – which we can definitely do with [create-meeting-artifacts](https://github.com/nodejs/create-node-meeting-artifacts) (the tool most folks use in Node.js for this).

I'll post more info for the meeting in about 1/2 an hour. See y'all soon! 🍻

**Comment by a0viedo** [comment]
I was wondering if it would make sense to remove (or possibly hide) from crowdin the directories of old documentation like v6.x or v8.x to help avoid confusion for incoming members of the Spanish translation group.

@alexandrtovmach is that possible?

**Comment by alexandrtovmach** [comment]
@a0viedo as a quick solution added mark "Low priority":
![image](https://user-images.githubusercontent.com/28801003/75775374-8c451980-5d5a-11ea-9ca3-b9093b0ac543.png)

I can permanently remove original files, but I'm not sure that's the good idea. Let me research a bit

**Comment by a0viedo** [comment]
Low priority would also help, deleting might be irreversible...if there was an alternative similar to hiding those directories I think it would be the best option.

**Comment by zeke** [comment]
Hey y'all! I am hoping to make it to this week's meeting on March 6.

Do we use the same zoom URL for each weekly meeting?

**Comment by obensource** [comment]
@zeke we do! Setting things up for tomorrow's meeting issue/minutes now. Would you mind sending me the email address you use for calendar invites via keybase or your preferred messaging method so I can invite you to the recurring event? Thanks a million! 🍻
